scripts/index.js

```diff
@@ -0,0 +1,281 @@
+// Open menu bar
+const openMenuBar = () => {
+  const menuBar = document.getElementById('the-menu-bar');
+  const menuBarOverlay = document.getElementById('menu-bar-overlay');
+  menuBar.addEventListener('click', () => {
+    menuBarOverlay.style.display = 'block';
+  });
+};
+openMenuBar();
+
+// Close menu bar
+function closeMenuBar() {
+  document.getElementById('menu-bar-overlay').style.display = 'none';
+}
+
+// Open mobile menu options
+const menuOptions = document.querySelectorAll('.menu-options');
+const mobileMenuOpts = () => {
+  menuOptions.forEach((menuOption) => {
+    menuOption.addEventListener('click', () => {
+      closeMenuBar();
+    });
+  });
+};
+mobileMenuOpts();
+
+// Creat and Display Projects Section
+
+const workSection = [
+  {
+    id: 1,
+    title: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featured_image: 'images/tonic_one.svg',
+    alt_message: 'My_Tonic_Project',
+    overview: ['CANOPY', 'Back End Dev', 2015],
+    overview_image: 'images/circle.svg',
+    technologies: ['html', 'css', 'javaScript'],
+    liveVersion: 'https://track92.github.io/Portfolio-mobile-version-skeleton/',
+    liveSource: 'https://github.com/TracK92/Portfolio-mobile-version-skeleton.git',
+  },
+  {
+    id: 2,
+    title: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featured_image: 'images/multi-stories.png',
+    alt_message: 'My_Multi_Stories_Project',
+    overview: ['CANOPY', 'Back End Dev', 2015],
+    overview_image: 'images/circle.svg',
```

Comment:
- Good design so far, PLease I noticed the popup for the desktop view those not fit properly on the screen, reason because the popup project size might be too big, please try to fix this.

##### Your design 

https://user-images.githubusercontent.com/58771507/174136107-95c55224-16b0-4b8f-9c92-1918557ca009.mp4

Reply:
Hi [Ginohmk](https://github.com/Ginohmk). Thank you for your feedback. It's nearly impossible to adjust the images the way that you want. I have tried as much as possible to alter the images but it looks stretched because those are the measurements of the images. If you have any suggestions on how this can be accomplished please let me know.

Reply:
Alright, please request for another review, so it will be approved.. I will like to join on a call with you tomorrow when you are free so we work on it okay. 

Please make a review request, so it's approved, since that's the only issue on your work.

Happy coding @TracK92

Reply:
I am your second reviewer, and I have seen your conversation, going to approve the project, but still suggest that the responsiveness of the popup card should be fixed. I assumed that you will fix it together.

Reply:
I would appreciate a call very much. I really tried working on it but I can only find those two options. I'll be available at any time tomorrow whenever you're ready. I have submitted my project for re-review.

Reply:
@ishimwezachee thanks .. 

@TracK92  please send me a message on slack tomorrow `@kanu mike chibundu` so we work on the pop-up responsiveness together ,okay.